### PR TITLE
Replaced the usage of asctime with strftime

### PR DESCRIPTION
**Pull request by RSilicon**
Replaced the usage of the obsolete asctime function with strftime for generating timestamp strings.

### CODE REVIEW

File: usr.bin/at/at.c [hunk 1]
```diff
@@ -880,8 +880,9 @@ main(int argc, char **argv)
 
 	if (atverify)
 	{
-	    struct tm *tm = localtime(&timer);
-	    fprintf(stderr, "%s\n", asctime(tm));
+	    char time_string[26];
+	    strftime(time_string, sizeof(time_string), "Www Mmm dd hh:mm:ss yyyy\n", localtime(&timer));
```
Comment (bsdimp) [inline usr.bin/at/at.c:884]: This makes no sense. Did you test these changes? That's not a valid format string.

**Review by bsdimp — CHANGES_REQUESTED** [review]
Please use valid strftime strings.
And please provide proof that this is working the same before/after.

**Comment by concussious** [comment]
Hi, I am a satisfied lock user. I use lock every single day. Thank you for your interest in lock. Please [don't break my heart](https://docs.freebsd.org/en/articles/committers-guide/#commit-log-message).

**Comment by jrtc27** [comment]
> Hi, I am a satisfied lock user. I use lock every single day. Thank you for your interest in lock. Please [don't break my heart](https://docs.freebsd.org/en/articles/committers-guide/#commit-log-message).

The first three sentences are spam. The final one links to some documentation that may be worth reading for the submitter, but the text used for the link is completely unrelated. This is extremely unhelpful and makes the entire message look like spam. Please do not do this again.

**Comment by concussious** [comment]
Copy, sorry, thank you.

**Pull request closed without merging**